invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunity.js

```diff
@@ -0,0 +1,31 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { Container } from "semantic-ui-react";
+import { Image } from "react-invenio-forms";
+
+export const RecordCommunity = (props) => {
```

Comment:
Why having a React component here and not going for Jinja?
Is it to re-use the `Image` cmp?

Reply:
Discussed IRL: agreed to go for Jinja